at_onboarding_flutter/lib/screen/at_onboarding_accounts_screen.dart

```diff
@@ -17,11 +17,14 @@ class AtOnboardingAccountsScreen extends StatefulWidget {
   /// the new atsign selected in the free atsign generator
   final String? newAtsign;
 
+  final OnboardingService onboardingService;
+
   const AtOnboardingAccountsScreen({
     Key? key,
     required this.atsigns,
     this.message,
     this.newAtsign,
+    required this.onboardingService,
```

Comment:
@sonle-geekyants We have added a new required parameter here, so, will this be a new major version release ?

Reply:
Yes, we need to increase major version